cns/restserver/nodesubnet.go

```diff
@@ -0,0 +1,59 @@
+package restserver
+
+import (
+	"context"
+	"net/netip"
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/cns/logger"
+	nodesubnet "github.com/Azure/azure-container-networking/cns/nodesubnet"
+	"github.com/Azure/azure-container-networking/cns/types"
+	errors "github.com/pkg/errors"
```

Comment:
I don't think you need the alias here

```suggestion
	"github.com/pkg/errors"
```